arangod/VocBase/LogicalView.cpp

```diff
@@ -198,18 +200,18 @@ Result LogicalView::instantiate(LogicalView::ptr& view, TRI_vocbase_t& vocbase,
 }
 
 Result LogicalView::rename(std::string&& newName) {
+  if (!ServerState::instance()->isSingleServer()) {
+    return {TRI_ERROR_CLUSTER_UNSUPPORTED};
+  }
+  // TODO thread unsafe
```

Comment:
@MBkkt I'm not sure what is the policies for leaving TODOs in arangodb code, just checking if this is expected to be fixed someday? maybe some ticket was created?

Reply:
It was, I just fixed race for cluster